DEVELOPMENT.md

```diff
@@ -532,6 +533,15 @@ $ git push origin master
 
 After pushing the tag, our CI system will start building the release. When this is done, you'll find a new release on the [releases page](https://github.com/TheThingsNetwork/lorawan-stack/releases). After this is done, you'll need to edit the release notes. We typically copy-paste these from `CHANGELOG.md`.
 
+When the CI system pushed the Docker image, it gets tagged as the current minor and patch version. If this release is not a backport but a latest stable release, you should manually tag and push `latest`:
+
+```bash
+$ versionDockerTag=${version#"v"} # v3.6.1 -> 3.6.1
```

Comment:
What about the major?

Reply:
That tag is deleted and we don't use it anymore